src/tests/evidence_scoring.py

```
@@ -0,0 +1,116 @@
+import json
+from difflib import SequenceMatcher
+
+with open('data/testing_data/sampled_claims_with_wiki.json', 'r', encoding='utf-8') as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)